docs/content/usage/packages/conan.en-us.md

```diff
@@ -30,12 +29,12 @@ conan remote add {remote} https://gitea.example.com/api/packages/{owner}/conan
 conan user --remote {remote} --password {password} {username}
 ```
 
-| Parameter  | Description |
-| -----------| ----------- |
-| `remote`   | The remote name. |
-| `username` | Your Gitea username. |
-| `password` | Your Gitea password. If you are using 2FA or OAuth use a [personal access token](development/api-usage.md#authentication) instead of the password. |
-| `owner`    | The owner of the package. |
+| Parameter    | Description                                                                                                                                    |
+| ------------ | ---------------------------------------------------------------------------------------------------------------------------------------------- |
+| `remote`   | The remote name.                                                                                                                               |
+| `username` | Your Gitea username.                                                                                                                           |
+| `password` | Your Gitea password. If you are using 2FA or OAuth use a[personal access token](development/api-usage.md#authentication) instead of the password. |
```

Comment:
```suggestion
| `password` | Your Gitea password. If you are using 2FA or OAuth use a [personal access token](development/api-usage.md#authentication) instead of the password. |
```